robot/description/urdf/constants.xacro

```diff
@@ -15,6 +15,12 @@
 
     <xacro:property name="chassis_link_mass" value="33.9"/>
 
+    <!-- Stand -->
+    <xacro:property name="stand_length" value="0.412"/>
```

Comment:
I don't believe we need the stand parameters ?

Reply:
It is used to compute a reasonable location for the ZED stereo cam on the stand. Plus there is no harm in keeping these constants, since they are meaningful to Caffeine in any case

Reply:
in case you need it for future reference, even if we don't reference it, would be nice to have